eo-runtime/src/test/java/EOorg/EOeolang/EOerrorTest.java

```diff
@@ -55,4 +63,64 @@ void makesToxicObject() {
         );
     }
 
+    @Test
+    void getsReadableError() {
+        ExAbstract error = null;
```

Comment:
@levBagryansky can we make this variable final? 
For example set it to `null` in `try` block and set to `exc` in `catch` block?

Reply:
@maxonfjvipon We cannot do it here, I get the error `Variable 'error' might already have been assigned to: 73`